README.md

```diff
@@ -316,255 +316,42 @@ public class LibertyRestEndpoint {
 2. Now create a file called LiberyRestEndpoint.java and add the following code to create the REST API calls for your second microservice......NEW CODE TO BE ADDED HERE!!!
 
 ## Creating the Web Application
-Now that we've created a gateway and microservices A and B, it's time to build a webapp to be able access and easily display the information within these microservices. 
+Now that we've created a gateway and two microservices (A and B), a webapp is needed to visualise the information in these microservices. To make this easier, a webapp has already been fully developed for this example. 
+The webapp used in this example has been created using Angular CLI and is running on a Node server. 
 
-1. Firstly we need to create a basic AngularJS webapp
-2. Then we need to use this basic webapp skeleton and customise it to show the information we wish to access from the microservices
-3. Finally we need to call API requests, through our gateway service, to the two microservices to be able to access and then display this information on the fron-end to the client.
+In order to use the webapp provided, it is essential that the pre-requisites previously stated are installed first. 
 
-In order to build our basic AngularJS webapp we'll be using the pre-requisites we've specified previously. Using node and npm we will install the @angular/cli tool which builds all of the files and installs all of the dependencies we will need for this webapp.
+1. Node and npm should already have been instaled. However @angular/cli also needs to be installed. To do this, first enter into the folder in which the webapp is to be placed (in this example enter the folder named *microservice-webapp*) using the terminal, enter the following command:
 
-In our terminal, enter into the folder in which you want the webapp. Then enter the command:
+    `npm install -g @angular/cli`
 
-  `npm install -g @angular/cli`
+    This may take a little while to run.
 
-This may take a little while to run.
+2. Next to avoid errors when switching branches commit all of the changes made so far to the module 1 branch of your fork. To do this enter the following commands in the terminal:
 
-After this, run the command:
+    i. `git add .`
 
-`ng new my-microprofile-app`
+    ii. `git checkout -m "enter comment here to describe what you are adding"`
 
-This will create the new AngularJS app file structure and place all of the downloaded files and dependencies within a folder calle "my-microprofile-app".
+    iii. `git push`
 
-Then enter into this folder:
 
-`cd my-microprofile-app`
+3. Now switch branches to module 2 to have access to the pre-created webapp. To switch branches enter the following command into the terminal:
 
-Then enter:
+    `git checkout module 2`

```

Comment:
Add the webapp code here so there is no need to switch branches as when they do the instructions will change!